caikit/runtime/train.py

```diff
@@ -80,6 +80,28 @@ def write_termination_log(text: str, log_file: str, enabled: bool):
         )
 
 
+# Final tasks before exiting the container
+def exit_complete(
+    exit_code: int,
+    save_path: str,
+    message: str,
+    termination_log_file: str,
+    enable_termination_log: bool,
+):
+    if exit_code != 0:
+        write_termination_log(message, termination_log_file, enable_termination_log)
+
+    if save_path:
+        try:
+            complete_path = os.path.join(save_path, ".complete")
+            log.info(f"Creating completion file at: {complete_path}")
```

Comment:
Lazy log interpolation here

```suggestion
            log.info("Creating completion file at: %s", complete_path)
```